gitlabform/gitlab/groups.py

```diff
@@ -1,7 +1,18 @@
+import functools
+import sys
+
+import cli_ui
+
+from gitlabform import EXIT_INVALID_INPUT
 from gitlabform.gitlab.core import GitLabCore, NotFoundException
 
 
 class GitLabGroups(GitLabCore):
+    @functools.lru_cache
```

Comment:
Check out the failed tests, @andrewjw . 
You need to add `()` for this code to work in Python 3.6 and 3.7.
Only in 3.8+ you can omit the parentheses if you don't pass any parameters.
(Btw: I learned that today. 😅 )

Reply:
I have to admit I only tested it on Python 3.8. Hopefully all the tests will pass now.